packages/form-js-viewer/src/render/components/form-fields/Fieldset.js

```diff
@@ -0,0 +1,41 @@
+import { useContext } from 'preact/hooks';
```

Comment:
What we also need to consider: is the dashed "children container" still what we want?

![image](https://user-images.githubusercontent.com/9433996/169809087-81a13c0c-4578-48bc-bf85-aa1443aeb7c9.png)

Reply:
This make me dilemmatic. Dash border looks verbosely, but it can indicates this area may drop-in. If we disallow fieldset nested, dash border may accepted.